CONTRIBUTING.rst

```diff
@@ -264,40 +254,48 @@ leverage multiple processes. Example usage:
 
    python -m pytest -n <NUMCORE> --cov pyvista
 
-Run all code examples in the docstrings. Be sure to export the
-``PYVISTA_OFF_SCREEN`` environmental variable to enable off-screen
-plotting.
+Documentation Testing
+~~~~~~~~~~~~~~~~~~~~~
+Run all code examples in the docstrings with:
 
 .. code:: bash
 
-   export PYVISTA_OFF_SCREEN=True
    python -m pytest -v --doctest-modules pyvista
 
-Run documentation testing by running:
+Build the documentation with:
 
 .. code:: bash
 
-   make
+   make -C doc html
```

Comment:
On Windows I have to use `python -msphinx -M html . _build` because Makefiles are not supported (as you mentioned here earlier). Maybe good to add such a note here as well now that everything else is made cross-platform?

Reply:
Added in b5fa2f74c97f4ed434919f5265f739ed0ad2e400